Products/PloneBooking/content/schemata.py

```diff
@@ -53,7 +53,7 @@
     StringField(
         'phone',
         default_method='getDefaultPhone',
-        required=False,
+        required=True,
```

Comment:
As I mentioned above, this is a bit heavy-handed; there should be another way for your other code to check for the presence of a phone number and handle it there without changing this here (you can imagine if the next contributor to this add-on decided to flip this back to required=False).